core/spec/lib/spree/core/validators/email_spec.rb

```diff
@@ -16,8 +22,19 @@ class Tester
     'valid-email@email.com',
     'valid_email@email.com',
     'validemail_@email.com',
-    'valid.email@email.com'
+    'valid.email@email.com',
```

Comment:
@vishalzambre can you please check for the following
'example-indeed@strangeexample.example'
'example-indeed@strangeexample.technology'.
Thanks in advance